x/ccv/types/events.go

```diff
@@ -2,17 +2,42 @@ package types
 
 // CCV events
 const (
-	EventTypeTimeout            = "timeout"
-	EventTypePacket             = "ccv_packet"
-	EventTypeChannelEstablished = "channel_established"
+	EventTypeTimeout                    = "timeout"
+	EventTypePacket                     = "ccv_packet"
+	EventTypeChannelEstablished         = "channel_established"
+	EventTypeFeeTransferChannelOpened   = "fee_transfer_channel_opened"
+	EventTypeConsumerClientCreated      = "consumer_client_created"
+	EventTypeAssignConsumerKey          = "assign_consumer_key"
+	EventTypeAddConsumerRewardDenom     = "add_consumer_reward_denom"
+	EventTypeRemoveConsumerRewardDenom  = "remove_consumer_reward_denom"
+	EventTypeSubmitConsumerMisbehaviour = "submit_consumer_misbehaviour"
+	EventTypeSubmitConsumerDoubleVoting = "submit_consumer_double_voting"
+	EventTypeExecuteConsumerChainSlash  = "execute_consumer_chain_slash"
+	EventTypeFeeDistribution            = "fee_distribution"
+	EventTypeConsumerSlashRequest       = "consumer_slash_request"
+	EventTypeVSCMatured                 = "vsc_matured"
 
-	AttributeKeyAckSuccess = "success"
-	AttributeKeyAck        = "acknowledgement"
-	AttributeKeyAckError   = "error"
-
-	AttributeChainID          = "chain_id"
-	AttributeValidatorAddress = "validator_address"
-	AttributeInfractionType   = "infraction_type"
-
-	AttributeValSetUpdateID = "valset_update_id"
+	AttributeKeyAckSuccess            = "success"
+	AttributeKeyAck                   = "acknowledgement"
+	AttributeKeyAckError              = "error"
+	AttributeValidatorConsumerAddress = "validator_consumer_address"
```

Comment:
Not related to this PR per se but I do not think `AttributeValidatorConsumerAddress` is used anywhere (?) in which case we can remove.

Reply:
Good catch. Removed.